virtual-desktop/src/app/plugin-manager/shared/plugin-manager.ts

```diff
@@ -25,20 +25,26 @@ export class PluginManager implements MVDHosting.PluginManagerInterface {
     this.loadApplicationPluginDefinitionsMap();
   }
 
+  clearApplicationPluginDefinitions(): void {
```

Comment:
Do same thing as https://github.com/zowe/zlux-platform/pull/42/files#r332638735 but in this PluginManager object too.